test/acceptance/pages/ProblemReportPage.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.pages
+
+import acceptance.conf.TestConfiguration
+import org.scalatestplus.selenium.Chrome.webDriver
+
+object ProblemReportPage extends BasePage {
+
+  val url: String = TestConfiguration.url("contact-frontend") + "/problem_reports_nonjs?service=pay"
+  val title = "Get help with a technical problem - GOV.UK"
+
+  val reportName: EmailField = emailField("report-name")
```

Comment:
I think these might need to be `def`s rather than vals. Being vals means that they will attempt to be evaluated on initialisation rather than in the specific test.